formfactory/models.py

```diff
@@ -215,9 +231,21 @@ def as_form(self, **kwargs):
                 "The model needs to be saved before a form can be generated."
             )
 
-        ordered_field_groups = self.fieldgroups.all().order_by(
-            "fieldgroupformthrough"
-        )
+        # Issue with order by and through, see:
```

Comment:
Ok now we really need to put this in a util somewhere. Too much copy and paste.

Reply:
Agreed, will give these another once over.